cluster-commands.go

```diff
@@ -1247,6 +1247,62 @@ func (adm *AdminClient) SiteReplicationResyncOp(ctx context.Context, site PeerIn
 	return res, err
 }
 
+// PoolEndpoints represent endpoints in a given pool
+// along with its setCount and setDriveCount.
+type PoolEndpoints struct {
+	// indicates if endpoints are provided in non-ellipses style
+	Legacy       bool      `json:"legacy"`
+	SetCount     int       `json:"setCount"`
+	DrivesPerSet int       `json:"drivesPerSet"`
+	Endpoints    Endpoints `json:"endpoints"`
+	CmdLine      string    `json:"cmdLine"`
+	Platform     string    `json:"platform"`
+}
+
+// EndpointServerPools - list of list of endpoints
+type EndpointServerPools []PoolEndpoints
+
+// Endpoints - list of same type of endpoint.
+type Endpoints []Endpoint
+
+// Endpoint - any type of endpoint.
+type Endpoint struct {
+	url.URL
```

Comment:
Maybe instead...

```suggestion
	Endpoint string `json:"endpoint"`
	Host string `json:"host"`	
```

To have the host and drive URL.

Reply:
@shtripat Then at least add it as a separate entity `UrlParts *url.URL json:"urlParts,omitempty"`